### ISSUE TITLE
Can't pass browser options with `driven_by`

### ISSUE THREAD

**Issue by d-mato** [rails/rails#37455]
### Steps to reproduce
```ruby
class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
  driven_by :selenium, using: :chrome do |driver_options|
    driver_options.add_argument('--disable-dev-shm-usage')
  end
end
```

### Expected behavior
```ruby
browser = ActionDispatch::SystemTesting::Browser.new(:chrome)
browser.capabilities.add_argument('--disable-dev-shm-usage')
browser.options # => <Selenium::WebDriver::Chrome::Options @args=#<Set: {"--disable-dev-shm-usage"}>, ...>
```

### Actual behavior
```ruby
browser = ActionDispatch::SystemTesting::Browser.new(:chrome)
browser.capabilities.add_argument('--disable-dev-shm-usage')
browser.options # => nil
```

Using not `using: :chrome` but `using: :headless_chrome`, browser#options returns correct value.

To pass browser options, Can't I use except `headless_chrome` or `headless_firefox ` as Browser argument ?

https://github.com/rails/rails/blob/6-0-stable/actionpack/lib/action_dispatch/system_testing/browser.rb#L23-L30

### System configuration
**Rails version**: 6.0

**Ruby version**: 2.6.4

**Comment by skinnyfit** [comment]
```ruby
driver_option.add_argument "disable-dev-shm-usage"
```
works for me (n.b. removal of leading `--`).

**Comment by d-mato** [comment]
Thanks your reply!

I tried with removing `--`, but not working.

```ruby
irb(main):012:0> browser = ActionDispatch::SystemTesting::Browser.new(:chrome)
=> #<ActionDispatch::SystemTesting::Browser:0x0000564ef93a89c0 @name=:chrome>
irb(main):013:0> browser.capabilities.add_argument('disable-dev-shm-usage')
=> #<Set: {"disable-dev-shm-usage"}>
irb(main):014:0> browser.options
=> nil
```

**Comment by skinnyfit** [comment]
You misread my comment. Remove the leading `--` from your initial snippet. So, in full:

```ruby
class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
  driven_by :selenium, using: :chrome do |driver_options|
    driver_options.add_argument('disable-dev-shm-usage')
  end
end
```

**Comment by Edouard-chin** [comment]
Will be fixed by https://github.com/rails/rails/pull/37792

**Comment by kamipo** [comment]
Fixed by #37792.
